projects/ADuCM3029_demo_cn0428_cn0429/src/adi_m355_gas_sensor.cpp

```diff
@@ -794,12 +776,12 @@ SENSOR_RESULT M355_GAS::ReadRload(uint8_t *pCfgData)
  */
 SENSOR_RESULT M355_GAS::ReadDataPPB(int32_t *pSensData)
 {
-	uint8_t pBuff[3] = { 0 };
+	uint8_t pBuff[4] = { 0 };
```

Comment:
As above, I would go with defined values wherever possible.

Reply:
This is not likely to change. This function reads 32bit variable from the I2C, so 4-byte array will always be needed.